packages/victory-errorbar/src/index.d.ts

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
+import {
+  EventPropTypeInterface,
+  StringOrNumberOrCallback,
+  VictoryCommonProps,
+  VictoryStyleInterface
+} from "victory-core";
+
+type dataType = {
+  x?: string | number;
+  y?: string | number;
+  errorX?: number | number[];
+  errorY?: number | number[];
+};
+
+export interface VictoryErrorBarProps extends VictoryCommonProps {
+  borderWidth?: number;
+  data?: dataType | dataType[];
+  errorX?: number | number[];
+  errorY?: number | number[];
+  events?: EventPropTypeInterface<"data" | "labels" | "parent", StringOrNumberOrCallback>[];
```

Comment:
the second argument for `EventPropTypeInterface` need to add `StringOrNumberOrCallback | string[]` so
`EventPropTypeInterface<"data" | "labels" | "parent", StringOrNumberOrCallback | string[]>[];` 

since `eventKey` type is the following ` string || integer || array[string] || function` https://formidable.com/open-source/victory/docs/victory-error-bar#eventkey